Makefile

```diff
@@ -71,4 +47,50 @@ endif
 ifndef HAS_GOLANGCI
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin
 endif
-	dep ensure
\ No newline at end of file
+	dep ensure
+
+################################################################################
+# Tests                                                                        #
+################################################################################
+
+.PHONY: lint
+lint:
+	golangci-lint run --config ./golangci.yml
+
+.PHONY: test
+test:
+	go test ./pkg/...
+
+################################################################################
+# Build / Publish                                                              #
+################################################################################
+
```

Comment:
Thoughts on adding meta `build` and `push` targets that resolve to `build-all-images` and `push-all-images`, respectively?  (Or, perhaps `docker-build` and `docker-push` for these...)

Reply:
Sure. And actually...

In `Brigade.js` there is a comment that indicates we build binaries and images in two separate jobs (and thus two separate make targets as well) because:

> // We build in a separate pod b/c AKS's Docker is too old to do multi-stage builds.

I know from recent experience (Drake) that this is no longer true. So is the issue that the particular _cluster_ we build on is so old that this is an issue? Even this seems unlikely...

If this _isn't_ an issue anymore, I think we can actually stop having separate "build" targets for binaries and images and just build both atomically.

Any thoughts?

Reply:
Thanks for the reminder!  Indeed, that comment is no longer valid.  We're on a more modern AKS cluster and we should definitely be able to introduce multi-stage builds.

Reply:
Good... already done in a recent commit. 😄

Reply:
Just perusing now -- so nice!